src/volume/volume_gaussian.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+	Lightmetrica - Copyright (c) 2019 Hisanari Otsu
+	Distributed under MIT license. See LICENSE file for details.
+*/
+
+#include <pch.h>
+#include <lm/core.h>
+#include <lm/volume.h>
+
+LM_NAMESPACE_BEGIN(LM_NAMESPACE)
+
+/*
+\rst
+.. function:: volume::gaussian
+
+    Gaussian volume.
+
+    :param color color: Stored color value.
+    :param float scalar: Stored max scalar value.
+    :param Vec3 pos: Center position of the volume
+    :param Vec3 sigma: Gaussian parameter of standard deviation for x,y,z axis
+\endrst
+*/
+class Volume_Gaussian : public Volume {
+private:
+	Bound bound_;// Computed from 0.001% of the max of gaussian, then times two
+	std::optional<Vec3> color_;
+	std::optional<Float> scalar_;
+	Vec3 pos_;
+	Vec3 sigma_;
+
+public:
+	LM_SERIALIZE_IMPL(ar) {
+		ar(bound_);
```

Comment:
Please add other member functions to this function for correct serialization (same for other classes).